Do these tests fail for you?

Hi, do these tests fail for you also?

```sh
=========================== short test summary info ============================
FAILED tests/test_sphinx_api.py::test_sphinx_api_1 - assert ('API\n'\n '***\n...
FAILED tests/test_sphinx_book.py::test_sphinx_book_text_cached - assert ('Fak...
FAILED tests/test_sphinx_book.py::test_sphinx_book_text_uncached - assert ('F...
======================== 3 failed, 101 passed in 6.84s =========================
```

```sh
=========================== short test summary info ============================
FAILED tests/test_sphinx_api.py::test_sphinx_api_1 - assert ('API\n'\n '***\n...
FAILED tests/test_sphinx_book.py::test_sphinx_book_text_cached - assert ('Fak...
FAILED tests/test_sphinx_book.py::test_sphinx_book_text_uncached - assert ('F...
======================== 3 failed, 101 passed in 6.84s =========================
```

--- a/.github/workflows/testing.yml
+++ b/.github/workflows/testing.yml
@@ -6,6 +6,8 @@ on:
   push:
     branches:
     - main
+  schedule:
+  - cron: "0 0 * * 0"
 
 jobs:
   cancel:

--- a/uqbar/book/console.py
+++ b/uqbar/book/console.py
@@ -1,6 +1,9 @@
 import code
 import inspect
 import itertools
+import os
+import sys
+import traceback
 from dataclasses import dataclass
 from typing import Any, Dict, List, Optional, Tuple
 
@@ -163,6 +166,20 @@ class Console(code.InteractiveConsole):
         for extension in self.extensions or []:
             extension.teardown_console(self)
 
+    ### PRIVATE METHODS ###
+
+    def _showtraceback(self):
+        sys.last_type, sys.last_value, last_tb = ei = sys.exc_info()
+        sys.last_traceback = last_tb
+        try:
+            self.write(
+                "".join(
+                    traceback.format_exception(ei[0], ei[1], last_tb.tb_next.tb_next)
+                )
+            )
+        finally:
+            last_tb = ei = None
+
     ### PUBLIC METHODS ###
 
     def flush(self) -> None:
@@ -213,7 +230,12 @@ class Console(code.InteractiveConsole):
         self.errored = True
 
     def showtraceback(self) -> None:
-        super().showtraceback()
+        if os.environ.get("CI"):
+            # GitHub Actions tracebacks include the code.py module
+            self._showtraceback()
+        else:
+            super().showtraceback()
+
         self.errored = True
 
     def write(self, string: str) -> None:

--- a/uqbar/book/console.py
+++ b/uqbar/book/console.py
@@ -173,9 +173,7 @@ class Console(code.InteractiveConsole):
         sys.last_traceback = last_tb
         try:
             self.write(
-                "".join(
-                    traceback.format_exception(ei[0], ei[1], last_tb.tb_next.tb_next)
-                )
+                "".join(traceback.format_exception(ei[0], ei[1], last_tb.tb_next))
             )
         finally:
             last_tb = ei = None
@@ -235,7 +233,6 @@ class Console(code.InteractiveConsole):
             self._showtraceback()
         else:
             super().showtraceback()
-
         self.errored = True
 
     def write(self, string: str) -> None:

--- a/uqbar/book/console.py
+++ b/uqbar/book/console.py
@@ -169,6 +169,12 @@ class Console(code.InteractiveConsole):
     ### PRIVATE METHODS ###
 
     def _showtraceback(self):
+        """
+        Re-implementation of code.InteractiveConsole's showtraceback().
+
+        No new functionality, but placing it here prevents the addition of
+        extra lines in the trace output when run under GHA.
+        """
         sys.last_type, sys.last_value, last_tb = ei = sys.exc_info()
         sys.last_traceback = last_tb
         try:

